developer-docs-site/docs/guides/move-guides/mint-nft-cli.md

```diff
@@ -3,76 +3,188 @@ title: "Mint an NFT with Aptos CLI"
 slug: "mint-nft-cli"
 ---
 
-# Mint an NFT
+# Mint an NFT with Aptos CLI
+
+This tutorial lets you use the Aptos CLI to mint NFTs in Aptos testnet so you can see how the process works and employ related functions in your code.
 
 ## Prerequisites
 
 This tutorial assumes you have:
 
-* a GitHub account
-* the GitHub CLI
-* the Aptos CLI (installed below)
+* a [GitHub account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account)
+* the [GitHub CLI](https://cli.github.com/)
+* the [Aptos CLI](../../cli-tools/aptos-cli-tool/install-aptos-cli.md) (or you can run from [aptos-core](https://github.com/aptos-labs/aptos-core) source via `cargo run`)
 
-## Mint with the Aptos CLI
+## Role of accounts
 
-Now that you are starting to write smart contracts with Move, let's create our first NFT with the Aptos CLI.
+When you are minting an NFT, the NFT is tied to your [account](../../concepts/accounts.md). Aptos ensures no one else can alter your collection of NFTs. This is why a private key is required to obtain signer capabilities. When you submit a transaction, you sign the transaction.
```

Comment:
nit: "the NFT is tied to your account" is ambiguous, because it could also mean that it's "tied" to your account and you cannot drop/transfer it? could we mention something like the NFT is being stored under your account address? 

another nit: I think `This is why a private key is required to obtain signer capabilities` is a bit ambiguous as well, because technically they are signing directly with the signer. Obtaining signer capability is a very specific term in our smart contracts, which usually means that we're getting the signer capability of an account for another account. 

`Aptos ensures no one else can alter your collection of NFTs.` - actually depending on how the token is set up, the creator might be able to burn the token without the token holder's permission. I think we're more ensuring that we intend to receive/mint a token in this case : )

Reply:
Thanks, @0xchloe . I removed the confusing text.